api-collections/bruno/psc-extensions-api/requests/healthcheck.bru

```diff
@@ -0,0 +1,12 @@
+meta {
+  name: Healthcheck
+  type: http
+  seq: 1
+}
+
+get {
+  url: {{baseUrl}}/actuator/health
```

Comment:
I don't think we need `actuator` in here. Typically we have `<app>/healthcheck`.